README.md

```diff
@@ -544,6 +550,24 @@ import { setRenderPartialOnly } from '@americanexpress/one-app-ducks';
 dispatch(setRenderPartialOnly(true));
 ```
 
+##### `setRenderTextOnly`
+
+Use this action creator to render text only from a holocron module, rather than a HTML.
+
+| Argument | Type | Description |
+|---|---|---|
+| `setTextOnly` | `Boolean` | (required) set whether to return text instead of HTML |
+| `tagReplacement` | `String` | Replace html tags with the character passed to this option i.e. '/n'. Defaults to empty string. |
+| `allowedTags` | `Array` | Comma separated list of HTML tags that are allowed to remain in the text response i.e. `['a','<strong>']`. Defaults to empty array. |
```

Comment:
Do you think it might be better to rename this to `htmlTagReplacement` and `allowedHtmlTags`?

Reply:
Kept the names from the strip tags library for consistency

Reply:
don't you think that context might not be know by users of this and adding the html to the text would help anyone using this intuitively know what it does?

Reply:
sure, renamed the variables https://github.com/americanexpress/one-app-ducks/pull/17/commits/6c186fabc4a4d3f5a624aca5810d3c4c0d8f1d76

Reply:
thank you